Feature/moneymong 79 소속 목록 UI

jhg3410 opened the pull request:
## 요약
소속 목록 UI 를 완성했습니다!!🙂

## 작업내용
|소속 존재 X|소속 존재 O|
|:----:|:----:|
|<img width="250" src="https://github.com/YAPP-Github/23rd-Android-Team-2-Android/assets/80373033/c5da148d-791f-44d9-a0c8-a4059aa37698">|<img width = "250" src = "https://github.com/YAPP-Github/23rd-Android-Team-2-Android/assets/80373033/970ff3e7-0679-40c2-b8dc-9922899be5c2">|

## 기타
- `tooltip` 은 추후 MDS  컴포넌트로 추가 예정입니다!
- `Icon` 을 일부 수정 및 추가했어요!
- `Yellow Color` 를 추가했어요!
- TextField 의 일부 attribute 를 수정했어요!
  - `ErrorState title`,  `countVisible`
- TextField 의 파라미터를 수정했어요!
![image](https://github.com/YAPP-Github/23rd-Android-Team-2-Android/assets/80373033/a63c294e-c0f2-4110-88da-4edc95528c10)
위와 같은 TextField 는 `singleLine = false` `minLines = 원하시는 line 수` 속성으로 지정하시면 됩니다!

jhg3410 committed:
moneymong-79 core: agency 모듈 생성

jhg3410 committed:
moneymong-79 feature: AgencySearchScreen UI 구현 (소속이 없을 경우)

jhg3410 committed:
moneymong-79 feature: AgencySearchScreen UI 구현 (소속이 존재할 경우)

jhg3410 committed:
moneymong-79 fix: icon, color 수정 및 추가

jhg3410 committed:
moneymong-79 fix: MDSTextField countVisible, titleColor(State == Eror) 변경

jhg3410 committed:
moneymong-79 fix: MDSTextField minLines 파라미터 추가, singleLine 기본값 제거

Heonbyeong commented on feature/agency/build.gradle.kts:
```diff
@@ -0,0 +1,20 @@
+
+android {
+    namespace = "com.moneymong.moneymong.feature.agency"
+    compileSdk = 33
```
compileSdk 충돌 가능성 있습니다 ! 삭제해주세요 🤓

jhg3410 replied:
엇!! 🧐

Heonbyeong commented on feature/agency/src/main/java/com/moneymong/moneymong/feature/agency/search/item/AgencyItem.kt:
```diff
@@ -0,0 +1,107 @@
+                maxLines = 1,
+                style = Body3
+            )
+            MDSTag(text = type.text, backgroundColor = Blue01, contentColor = Blue04)
```
파라미터가 여러 개일 경우, 줄바꿈 해주세요 ! (컨벤션 맞추기 위함)

jhg3410 replied:
예압!🙂🙂

Heonbyeong commented on feature/agency/src/main/java/com/moneymong/moneymong/feature/agency/search/AgencySearchScreen.kt:
```diff
@@ -0,0 +1,122 @@
+            contentDescription = "agency image",
+        )
+        Text(
+            text = "아직 등록된 소속이 없어요\n" +
```
개인적으로 텍스트는 한 줄로 표현하는게 가독성이 더 높은 것 같습니다
`text = "아직 등록된 소속이 없어요\n하단 버튼을 통해 등록해보세요"`

jhg3410 replied:
좋아요~👍👍

Heonbyeong reviewed (approved):
고생하셨습니다 🐌

jhg3410 committed:
moneymong-79 fix: 병헌님 리뷰 반영

https://github.com/YAPP-Github/23rd-Android-Team-2-Android/pull/17#pullrequestreview-1776406377

eunseo0105 reviewed (approved):
고생하셨습니다~
